docs/integrations/heroku-rails.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: docs
+title: "Heroku with Rails - Integrations"
+---
+
+{% include helpers/reading_time.html %}
+
+##### Integrations
+
+# Heroku with Rails
+
+Learn how to make Heroku, Rails, and Dotenv Vault work together. This tutorial assumes you have already created a `.env` file and [synced it](/docs/tutorials/sync).
+
+## 1. Set up Rails app for Heroku
+
+Set up your Rails app to work with Heroku.
+
+Example shows using environment variable [example](https://github.com/dotenv-org/integration-example-heroku-rails/blob/374e9a3e5e5f6ffe2f4a83f08bf2c0222871ed40/app/views/welcome/index.html.erb)
+
+```
+// Add 'dotenv-vault-rails' to Gemfile
+gem 'dotenv-vault-rails'
+```
+
+[example](https://github.com/dotenv-org/integration-example-heroku-rails/blob/374e9a3e5e5f6ffe2f4a83f08bf2c0222871ed40/Gemfile#L4)
+
+Run bundle install.
+
+```
+$ bundle install
+```
+
+## 2. Require dotenv-vault-ruby
+
+Require [dotenv-vault-ruby](https://github.com/dotenv-org/dotenv-vault-ruby).
```

Comment:
Change this to just Require [dotenv-vault](https://github.com/dotenv-org/dotenv-vault-ruby)

The -ruby is only there because dotenv-vault is already taken on our organization.